src/components/UploadCSVView/ReadOnlyRow/ReadOnlyRow.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Td, Tr, useDisclosure } from '@chakra-ui/react';
+import { CheckIcon, CloseIcon, DeleteIcon, EditIcon } from '@chakra-ui/icons';
+import DeleteBoxModal from '../DeleteBoxModal/DeleteBoxModal';
+import styles from './ReadOnlyRow.module.css';
+
+const ReadOnlyRow = ({ data, editRow, handleDeleteRow, isError }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
```

Comment:
Same here, it's better to be more specific in case you need to add similar variables/functionality later on :)